_pages/2019/x/psets/8/finance/finance.adoc

```diff
@@ -90,6 +90,24 @@ Last up is `layout.html`. It's a bit bigger than usual, but that's mostly becaus
 
 == Specification
 
+=== Configuring
+
+Before getting started on this assignment, we'll need to register for an API key in order to be able to query IEX's data. To do so, follow these steps:
+
+. Visit https://iexcloud.io/cloud-login#/register/[iexcloud.io/cloud-login#/register/].
```

Comment:
I think we usually omit the protocol in URLs. Did I tweak this correctly, @brianyu28?